import PureCloudPlatformClientV2
from PureCloudPlatformClientV2.rest import ApiException
from pprint import pprint

# Configure OAuth2 access token for authorization: PureCloud OAuth
PureCloudPlatformClientV2.configuration.access_token = 'your_access_token'
# or use get_client_credentials_token(...), get_saml2bearer_token(...), get_code_authorization_token(...) or get_pkce_token(...)

# create an instance of the API class
api_instance = PureCloudPlatformClientV2.OAuthApi();
scope_id = 'scope_id_example' # str | Scope ID
accept_language = ''en-us'' # str | The language with which to display the scope description. (optional) (default to 'en-us')

try:
    # An OAuth scope
    api_response = api_instance.get_oauth_scope(scope_id, accept_language=accept_language)
    pprint(api_response)
except ApiException as e:
    print("Exception when calling OAuthApi->get_oauth_scope: %s\n" % e)